myosuite/tests/test_tutorials.sh

```diff
@@ -22,7 +23,9 @@ declare -a StringArray=(
                          "2_Load_policy.ipynb" \
                          "3_Analyse_movements.ipynb" \
                         #  "4_Train_policy.ipynb" \
```

Comment:
Some of the tutorials are commented out. Let's make sure they pass CIs

Reply:
@vikashplus the mjrl is broken. Can you please check what is not working?
@P-Schumacher the Deprl tutorial is broken. Can you please check it?